src/uct/api/uct.h

```diff
@@ -1558,7 +1559,7 @@ ucs_status_t uct_config_get(void *config, const char *name, char *value,
 
 /**
  * @ingroup UCT_CONTEXT
- * @brief Modify interface/MD configuration.
+ * @brief Modify interface/memory domain/connection manager configuration.
```

Comment:
I suggest to change it from ```interface/memory domain/connection manager``` ->
```interface, memory domain, or connection manager```, where each one is ```@ref``` to correspond object in UCT.